README.md

```diff
@@ -4,59 +4,86 @@
 
 # Gitbase Playground
 
-Web application to query git repositories using SQL. Powered by [gitbase](https://github.com/src-d/gitbase).
+Web application to query Git repositories using SQL. Powered by [gitbase](https://github.com/src-d/gitbase), it allows to perform SQL queries about the Git history and the [Universal AST](https://doc.bblf.sh/) of the code itself.
 
 ![Screenshot](.github/screenshot.png?raw=true)
 
-
 # Usage
 
-## Dependencies
+## With Docker Compose
+
+The easiest way to run Gitbase Playground and its dependencies is using [Docker Compose](https://docs.docker.com/compose/install/).
 
-The playground will run the queries against a [gitbase](https://github.com/src-d/gitbase) server, and will request UASTs to a [bblfsh](https://doc.bblf.sh/) server; both should be accessible for the playground; you can check its default [configuration values](docs/CONTRIBUTING.md#configuration).
+The first step is to populate a directory with some Git repositories to be served by gitbase before running it. For example:
 
+```bash
+$ mkdir $HOME/repos
+$ cd $HOME/repos
+$ git clone git@github.com:src-d/gitbase.git
+$ git clone git@github.com:bblfsh/bblfshd.git
+$ git clone git@github.com:src-d/gitbase-playground.git
+```
 
-## Run the Playground
+Next you will need to download the `docker-compose.yml` file from this repository and run `docker-compose`. This tool will run three different containers: the playground frontend itself, gitbase, and bblfshd. To kill the running containers use `Ctrl+C`.
+
+```bash
+$ wget https://raw.githubusercontent.com/src-d/gitbase-playground/master/docker-compose.yml
+$ docker-compose pull
+$ GITBASEPG_REPOS_FOLDER=$HOME/repos docker-compose up --force-recreate
+```
 
-You can run the app from a docker image, a released binary or installing and building the project.
+The server should be now available at http://localhost:8080.
```

Comment:
Why not making it a hyperlink?

Reply:
It is a hyperlink if you see it rendered.

Reply:
Changed in 3a37a77